src/main/java/org/springframework/metrics/instrument/scheduling/ExecutorServiceMetrics.java

```diff
@@ -0,0 +1,31 @@
+package org.springframework.metrics.instrument.scheduling;
+
+import org.springframework.metrics.instrument.MeterRegistry;
+import org.springframework.metrics.instrument.binder.MeterBinder;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+
+public class ExecutorServiceMetrics implements MeterBinder {
+
+  private final String name;
+  private final ExecutorService executorService;
+
+  public ExecutorServiceMetrics(String name, ExecutorService executorService) {
+    this.name = name;
+    this.executorService = executorService;
+  }
+
+  @Override
+  public void bindTo(MeterRegistry registry) {
+    if(executorService instanceof ThreadPoolExecutor) {
+      ThreadPoolExecutor tp = (ThreadPoolExecutor) executorService;
+      registry.gauge(name+"_scheduled_total", tp, ThreadPoolExecutor::getTaskCount); //Really should be a counter
+      registry.gauge(name+"_completed_total", tp, ThreadPoolExecutor::getCompletedTaskCount); //Really should be a counter
```

Comment:
In this case we can assume the guarantee since it is documented to always go up in the ThreadPoolExecutor.

The root problem may be that we need a way to create custom collectors against the spring-metrics api instead of bloating the `Registry` API? 

Would that be a problem for Spectator metrics?